adafruit_is31fl3731/__init__.py

```diff
@@ -341,22 +397,20 @@ def pixel(self, x, y, color=None, blink=None, frame=None):
 
     # pylint: enable-msg=too-many-arguments
 
-    def image(self, img, blink=None, frame=None):
+    def image(self, img: bytes = None, blink: bool = False, frame: int = 0) -> bytes:
```

Comment:
in this one `img` should be `Image` that comes from an import like `from PIL import Image` I believe.

I think `frame` should keep it's old default of `None` as well since there is logic in the functions were it will end up checking for that. That would also make it `Optional[int]`